### DISCUSSION TITLE
旧版网页关闭了，怎么刷题

### DISCUSSION THREAD

**Discussion by 333-cell** [11273/mooc-work-answer#166] [Q&A]
旧版智慧职教网页已经关闭了，账号登不上了

**Comment by 11273** [comment]
https://mooc-old.icve.com.cn/ 可以进去呢

**Reply by 333-cell** [reply]
但是账号密码填对了，它显示错误

**Reply by 11273** [reply]
那可能不能刷了吧，可能后面都会成新版了

**Reply by 333-cell** [reply]
那脚本也会更新吗

**Reply by 11273** [reply]
会根据需求调整的，现在有新版，要是旧版停了那也停了，没停的话还会更新

**Reply by 333-cell** [reply]
![6f8f69d7-3b1c-4268-ab8e-05322bddd96d](https://github.com/user-attachments/assets/a6d991e4-6df7-429c-b9ff-6281cec05afe)
是我账号注册晚了吗 现在进不去

**Reply by 11273** [reply]
能登陆官方才可以登录，是用的官方接口，也是接口返回的信息的
